rust-mode.el

```diff
@@ -170,7 +170,18 @@ function or trait.  When nil, where will be aligned with fn or trait."
   "Face for the question mark operator."
   :group 'rust-mode)
 
+(defface rust-builtin-formatting-macro-face
+  '((t :inherit font-lock-builtin-face))
+  "Face for builtin formatting macros \(print! &c.\)."
+  :group 'rust-mode)
+
+(defface rust-string-interpolation-face
+  '((t :slant italic :inherit font-lock-string-face))
```

Comment:
I figure it’s better to keep the default styling conservative and let people customize it if they need something more dramatic. That said, the italics have been working really well for me!